pkg/apis/extensions/v1alpha1/types_operatingsystemconfig.go

```diff
@@ -223,7 +223,9 @@ type CRIName string
 
 const (
 	// CRINameContainerD is a constant for ContainerD CRI name
-	CRINameContainerD = "containerd"
+	CRINameContainerD CRIName = "containerd"
```

Comment:
`test/integration/shoots/operations/containerruntime.go` needs to be adapted as it is using `CRINameContainerD` to assert the node the Node label `worker.gardener.cloud/cri-name` value.

```
• Failure [277.286 seconds]
Shoot container runtime testing
/Users/i331370/go/src/github.com/gardener/gardener/test/integration/shoots/operations/containerruntime.go:30
  [DEFAULT] [SERIAL] [SHOOT] should add worker pool with containerd [It]
  /Users/i331370/go/src/github.com/gardener/gardener/test/framework/gingko_utils.go:26

  Expected
      <string>: containerd
  to equal
      <v1alpha1.CRIName>: containerd

  /Users/i331370/go/src/github.com/gardener/gardener/test/integration/shoots/operations/containerruntime.go:98
```